packages/components/src/components/CoinLogo/index.native.tsx

```diff
@@ -31,12 +31,20 @@ const Logo = styled.Image``;
 
 interface Props {
     network: string;
+    height: number | string;
 }
 
-const CoinLogo = ({ network, ...rest }: Props) => {
+const CoinLogo = ({ network, height = 50, ...rest }: Props) => {
```

Comment:
I would drop the default value to keep it consistent with a web component

Reply:
I would rather add the default prop in web component, so it will have some consistent default size because now if you don't specify the size the logos look weird and each have different size.

Reply:
where it looks weird? 
You can set the default size in application (which everyone will do anyway). I don't see any advantages in picking a random value as the default height. It doesn' make sense to me since there is nothing like the most used size, even in storybook you are using a custom height, different than this default (and in the wallet we also use different height).

Reply:
I think the most commonly used sizes for logos/icons are 16x16, 32x32, 64x64, ...
I'm looking at the logo images in components/src/images/coins/ and i think we should unify the size's of these images to 64x64. (because now they have all different sizes).
In react native you need to set the size otherwissse there is nothing rendered (renders 0x0px element) so there needs to be some default size.
So the default size of the image in react native would be 64 (`height = 64`, or maybe call it `size`?) and in react the default size is unset so it's rendered as 64x64 too.

Reply:
I am fine with size unification. I can even understand that the image won't render without specifying the dimensions. If that's how the react-native Image component works then it's fine with me and I think it's up to the dev to use it properly inside his app (suite/storybook/...).imo in most situations, we won't use the defaults anyway (as I mentioned earlier we currently use different sizes), but if it helps anybody then let's pick 32 or 64 (something smallish).